src/snowcli/cli/common/cli_global_context.py

```diff
@@ -184,6 +188,10 @@ def connection_context(self) -> _ConnectionContext:
     def connection(self) -> SnowflakeConnection:
         return self.connection_context.connection
 
+    @property
```

Comment:
Can we move this property to be placed above its setter method (to keep the same convention as other properties with setters)?